payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -47,7 +39,10 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
     internal var viewModelFactory: ViewModelProvider.Factory =
         PaymentSheetViewModel.Factory(
             { application },
-            { requireNotNull(starterArgs) }
+            { requireNotNull(starterArgs) },
+            ActivityLauncherFactory.ActivityHost(this),
+            { AuthActivityStarter.Host.create(this) },
```

Comment:
As discussed during 1:1, we can't pass an Activity instance to the ViewModel's constructor.

Reply:
With #3834 no longer requires [Activity]

Reply:
With #3834 it no longer requires an [Activity]